yarn.lock

```diff
@@ -4933,11 +4933,16 @@ aws4@^1.8.0:
   resolved "https://registry.yarnpkg.com/aws4/-/aws4-1.9.1.tgz#7e33d8f7d449b3f673cd72deb9abdc552dbe528e"
   integrity sha512-wMHVg2EOHaMRxbzgFJ9gtjOOCrI80OHLG14rxi28XwOW8ux6IiEbRCGGGqCtdAIg4FQCbW20k9RsT4y3gJlFug==
 
-axe-core@^3.3.2, axe-core@^3.5.3:
+axe-core@^3.3.2:
   version "3.5.3"
   resolved "https://registry.yarnpkg.com/axe-core/-/axe-core-3.5.3.tgz#5b7c0ee7c5197d546bd3a07c3ef701896f5773e9"
   integrity sha512-HZpLE7xu05+8AbpqXITGdxp1Xwk8ysAXrg7MiKRY27py3DAyEJpoJQo1727pWF3F+O79V3r+cTWhOzfB49P89w==
 
+axe-core@^4.0.2:
+  version "4.0.2"
+  resolved "https://artifactory.acorn.cirrostratus.org/artifactory/api/npm/npms/axe-core/-/axe-core-4.0.2.tgz#c7cf7378378a51fcd272d3c09668002a4990b1cb"
```

Comment:
You've installed this packed via a private Artifactory instance as opposed to NPM's public registry. Can you please update this, and rebase your branch against `master`.

If this really is a non-API breaking change, I'd really love to see this make it into the repo. Thanks so much for the contribution!

Reply:
Womp 😛 nice catch! Forgot to change the registry to NPM. Will get that updated!